package.json

```diff
@@ -48,31 +48,32 @@
         "verify": "node scripts verify",
         "version": "node scripts/jekyll-config && npm run build:storybook && npm run build && git add -A dist docs",
         "transpile": "babel docs/src/js --out-dir docs/_babel",
-        "snapshots": "npx gulp runSnapshots --stories $STORIES",
-        "snapshots:dry": "npx gulp runSnapshots --dry yes --stories $STORIES",
-        "snapshots:execute": "percy storybook ./docs/storybook --include",
-        "snapshots:execute:dry": "percy storybook ./docs/storybook --dry-run --include",
-        "snapshots:all": "percy storybook ./docs/storybook --include '.*'",
-        "snapshots:all:dry": "percy storybook --dry-run ./docs/storybook --include '.*'",
+        "build:snapshots": "npm run copy:assetsToStorybook && cross-env STORYBOOK_DS=6 build-storybook -o ./.storybook-percy",
+        "snapshots": "npm run build:snapshots && npx gulp runSnapshots --stories $STORIES",
+        "snapshots:dry": "npm run build:snapshots && npx gulp runSnapshots --dry yes --stories $STORIES",
```

Comment:
I didn't catch this before, but I don't think we need to build storybook for dry runs since we're only seeing the list of snapshots to be taken and it's not doing an actual build. It's trivial and shouldn't hold up the release by any means, but we should probably adjust it.